ScnSoft-Education/LeetCodeTraining/LongestSubstring/LongestSubstringSolution.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeetCodeTraining.LongestSubstring
+{
+    //https://leetcode.com/problems/longest-substring-without-repeating-characters/
+    public static class LongestSubstringSolution
+    {
+        public static int LengthOfLongestSubstring(string s)
+        {
+            if (s.Length == 0)
+            {
+                return 0;
+            }
+
+            if (s.Length == 1)
+            {
+                return 1;
+            }
+
+            StringBuilder longestString = new StringBuilder();
```

Comment:
It seems longestString is redundant and we can just use variable like int longestStringLength and calculate it's value and set to 0 when it is required.